### ISSUE TITLE
[Bug]: Agent stuck initialize

### ISSUE THREAD

**Issue by PierrunoYT** [OpenDevin/OpenDevin#1308]
### Is there an existing issue for the same bug?

- [X] I have checked the troubleshooting document at https://github.com/OpenDevin/OpenDevin/blob/main/docs/guides/Troubleshooting.md
- [X] I have checked the existing issues.

### Describe the bug

I run the docker command with the main tag but when I go to localhost:3000 the agent stuck initialize

### Current Version

```bash
Main Version
```

### Installation and Configuration

```bash
docker run \
    -e LLM_API_KEY \
    -e WORKSPACE_MOUNT_PATH=$WORKSPACE_DIR \
    -v $WORKSPACE_DIR:/opt/workspace_base \
    -v /var/run/docker.sock:/var/run/docker.sock \
    -p 3000:3000 \
    ghcr.io/opendevin/opendevin:main
```

### Model and Agent

Not sure tho

### Reproduction Steps

Run the Docker Command with the main tag instead of 0.3.1

### Logs, Errors, Screenshots, and Additional Context

pierruno@Pierruno:/mnt/c/Users/pierr/OpenDevin$ docker run \
    -e LLM_API_KEY \
    -e WORKSPACE_MOUNT_PATH=$WORKSPACE_DIR \
    -v $WORKSPACE_DIR:/opt/workspace_base \
    -v /var/run/docker.sock:/var/run/docker.sock \
    -p 3000:3000 \
    ghcr.io/opendevin/opendevin:main
Unable to find image 'ghcr.io/opendevin/opendevin:main' locally
main: Pulling from opendevin/opendevin
13808c22b207: Already exists
6c9a484475c1: Already exists
78bef5c7424f: Already exists
42f0d54f5caa: Already exists
1723cff2f16b: Already exists
9dbd87b6acd5: Pull complete
4fb59a446c3c: Pull complete
ab27f25f739f: Pull complete
569b7cc45339: Pull complete
5103560c915a: Pull complete
3d6c6d620357: Pull complete
ee3c2cd71fcb: Pull complete
eb42e9889b68: Pull complete
Digest: sha256:4d0183c3a6c35f591f14a445c8ee186fad8a9b4f57d4a9fbef6c8d0eb4232772
Status: Downloaded newer image for ghcr.io/opendevin/opendevin:main
INFO:     Started server process [1]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:3000 (Press CTRL+C to quit)
INFO:     172.17.0.1:36482 - "GET / HTTP/1.1" 307 Temporary Redirect
INFO:     ('172.17.0.1', 36494) - "WebSocket /ws?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzaWQiOiJiMWU1ZDM1My05MDM1LTRiYTEtOGEwMC04NGZmNDU0ZWMxYTEifQ.VHFX1AwnLJprhuTP_CHGpOpUBZNlruNT6Guzt1y2G9k" [accepted]
INFO:     connection open
Starting loop_recv for sid: b1e5d353-9035-4ba1-8a00-84ff454ec1a1
INFO:     172.17.0.1:36498 - "GET /api/refresh-files HTTP/1.1" 200 OK
INFO:     172.17.0.1:36498 - "GET /api/configurations HTTP/1.1" 404 Not Found
INFO:     172.17.0.1:36512 - "GET /api/messages/total HTTP/1.1" 200 OK
INFO:     172.17.0.1:36498 - "GET /sw.js HTTP/1.1" 404 Not Found
INFO:     172.17.0.1:36512 - "GET /workbox-c06b064f.js HTTP/1.1" 404 Not Found

**Comment by rbren** [comment]
@PierrunoYT the command you're running isn't the latest command from the README

**Comment by PierrunoYT** [comment]
> @PierrunoYT the command you're running isn't the latest command from the README

I tried main and  0.3.1 and both did not worked for me.

**Comment by enyst** [comment]
@PierrunoYT have you made it work now? Please note that we have released another version, 0.4.0.

If not, can you post the error you get with 0.4.0?

**Comment by PierrunoYT** [comment]
I need to check again. Did not used OpenDevin for the last few day because I still have some problems to get it fully working while chatting.
